packages/docz-tools/src/gatsby-theme-docz/components/index.ts

```diff
@@ -0,0 +1,29 @@
+// @ts-expect-error
```

Comment:
Want to look at these together?

Reply:
what are your thoughts on a follow up for this?

Reply:
I'm fine with that if you wanna make a task for us to do so.